update first resources to include cma (#5183)
centreon/centreon-documentation: cloud/getting-started/monitoring.md

---
id: monitoring

> title: Monitoring your first host

Reviewer: harmonisation EN/FR. pourquoi ne pas garder la question?

> title: Monitoring your first host

Author: ```suggestion
title: How do I monitor my first host?
```

---

Make sure you have read [Centreon basics](concepts.md) before monitoring your first host.

## Monitoring a host

> > We recommend installing the [Centreon Monitoring Agent (CMA)](../cma/cma.md), a piece of software that monitors the host it is installed on, collecting metrics and computing statuses to send to Centreon. When you install CMA, a host is automatically created for it.

Reviewer: Only if you have selected the option, and only for 1 direction 'check which one).

> > We recommend installing the [Centreon Monitoring Agent (CMA)](../cma/cma.md), a piece of software that monitors the host it is installed on, collecting metrics and computing statuses to send to Centreon. When you install CMA, a host is automatically created for it.

Author: ```suggestion
> We recommend installing the [Centreon Monitoring Agent (CMA)](../cma/cma.md), a piece of software that monitors the host it is installed on, collecting metrics and computing statuses to send to Centreon. When you install CMA, you can choose to have a host automatically created for it.
```

> > We recommend installing the [Centreon Monitoring Agent (CMA)](../cma/cma.md), a piece of software that monitors the host it is installed on, collecting metrics and computing statuses to send to Centreon. When you install CMA, you can choose to have a host automatically created for it.

Reviewer: ```suggestion
> We recommend installing the [Centreon Monitoring Agent (CMA)](../cma/cma.md), a piece of software that monitors the host it is installed on, collecting metrics and computing statuses to send to Centreon. When you install CMA on a host, you can choose to have this host automatically created in Centreon.
```

To monitor a host in Centreon Cloud:

> 1. Create the host with the **Add** button in **Configuration** > **Hosts** > **Hosts**.

Reviewer: ```suggestion
1. To create a host, go to **Configuration** > **Hosts** > **Hosts** and then click **Add**.
```

2. Link the new host to the poller you want it to be monitored by.
3. Link the new host to the correct template provided by a [Monitoring Connector](../monitoring/pluginpacks.md).

> 4. [Export the configuration](../monitoring/monitoring-servers/deploying-a-configuration.md). The resource will appear in the **Monitoring > Resources status** page shortly, where you can track any changes in status.

Reviewer: ```suggestion
4. [Export the configuration](../monitoring/monitoring-servers/deploying-a-configuration.md). The host will appear in the **Monitoring > Resources status** page shortly, where you can track any changes in status.
```
